.scripts/localeValidate.js

```diff
@@ -65,17 +68,35 @@ fs.readdir(defaultPath, (error, files) => {
       fs.readFileSync(join(defaultPath, file)).toString()
     );
 
-    languages.forEach((lng) => {
+    languagesButEn.forEach((lng) => {
       const otherPath = join(localeDir, lng);
       const otherJson = JSON.parse(
         fs.readFileSync(join(otherPath, file)).toString()
       );
 
-      const a = getDeepKeys(defaultJson);
-      const b = getDeepKeys(otherJson);
+      const en = getDeepKeys(defaultJson);
+      const others = getDeepKeys(otherJson);
+      for (i in Object.values(en)) {
```

Comment:
None of this is necessary, this is why:

We already have a function from `localeOrderKeys` that orders the entire file the way we want. So we do not need to introduce any more logic other than this:
- **re-use** the function from `localeOrderKeys` to get the correct ordered JSON.
-  compare the **unordered** json to the ordered JSON: `json === ordered(json)`.
- If they match, file is ordered, else it is not ordered.